tests/integration/utils/mock-api.js

```diff
@@ -38,6 +38,12 @@ export const startMockApi = ({ routes, silent }) => {
     })
   })
 
+  app.get('/site/site_id/integrations/safe', function onRequest(req, res) {
```

Comment:
We fetch integrations from a different endpoint, so passing this url in `routes` above won't work.

https://github.com/netlify/build/blob/main/packages/config/src/api/site_info.ts#L129-L131